### ISSUE TITLE
The last line in the markdown file is removed after converted.

### ISSUE THREAD

**Issue by bohrshaw** [wikimatze/hammer.vim#55]

**Comment by andrewferrier** [comment]
Agreed, I can recreate this issue too. The last line of every markdown file I preview in vim using :Hammer is removed from the generated output. The workaround is to add a blank line as the last line (as I see you are doing in the tests!).
